jekyll/_cci2/server/air-gapped-installation/phase-2-configure-object-storage.adoc

```diff
@@ -0,0 +1,105 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 2 - Configuring Object Storage
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to configure object storage through Min.IO to run CircleCI Server in an air-gapped environment.
+:icons: font
+:toc: macro
+:toc-title:
+
+[#configure-minio]
+== Initial Min.IO Setup
+
+[#create-airgap-data-bucket]
+=== 1. Create a airgap-data bucket.
+Create a new bucket in min.io named `airgap-data`. This bucket will be for your values.yaml file.
+
+[#create-circle-data-bucket]
+=== 2. Create a circle-data bucket.
+Create a new bucket in min.io named `circle-data`.
+
+[#configure-circle-data-bucket]
+=== 3. Configure the circle-data bucket.
```

Comment:
```suggestion
=== 2. Create a circleci-data bucket.
Create a new bucket in min.io named `circleci-data`.

[#configure-circle-data-bucket]
=== 3. Configure the circleci-data bucket.
```

Reply:
Waiting on confirmation before changing this:
https://circleci.atlassian.net/wiki/spaces/server/pages/6650560666/Test+Case+-+Air+Gap

Reply:
I am missing something, confirmation of what?